### PR TITLE
KIALI-1738 Support app without workload

### PR DESCRIPTION
**Pull request by lucasponce**
Counterpart of https://github.com/kiali/kiali/pull/564 to support in the UI applications with no workloads.
(Corner case when a workload is a simple pod).

### CODE REVIEW

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#218](https://paste.fedoraproject.org/paste/oMOqsP3AsVT-mkNtXDt2IQ)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1624]](https://paste.fedoraproject.org/paste/9NRM4-L79LIxmyZs9PwvuA)
  * :heavy_check_mark: Istio-Upstream [#[472]](https://paste.fedoraproject.org/paste/x9Scdg4FQPSpwUkGO16G8A)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1812, 1814]](https://paste.fedoraproject.org/paste/d6H7Rl-rPJd7DI2POn2KDg)
  * :heavy_check_mark: install-bookinfo [#[999]](https://paste.fedoraproject.org/paste/DgeWLO3xKFsfSjhz7vEPLg)
  * :heavy_check_mark: Kiali-on-Openshift [#[1014]](https://paste.fedoraproject.org/paste/7TTYNWuxYcVYNGybpTxQNw)
  * :red_circle: run-kiali-e2e-tests [#[421]](https://paste.fedoraproject.org/paste/~903kkaTS5eYcDWKXoPUhw)


KIALI_NAME=kialiuipr751-istio-system

File: src/pages/AppDetails/AppInfo/AppDescription.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ class AppDescription extends React.Component<AppDescriptionProps, AppDescription
   }
 
   istioSidecar() {
-    let istioSidecar = true;
+    let istioSidecar = this.props.app.workloads && this.props.app.workloads.length > 0 ? true : false;
```
Comment (jotak) [inline src/pages/AppDetails/AppInfo/AppDescription.tsx]: (not for this PR but...) it's strange that we duplicate the same algorithm, to check for sidecar on the backend side for app list, and on the frontend side for app details, is there a reason for that?

Comment (lucasponce) [reply src/pages/AppDetails/AppInfo/AppDescription.tsx]: Oh, true, the istioSide car check is done in the backend for the list, but it was not added in the details of the app.
I can change it on both (backend/frontend).
